### ISSUE TITLE
How to contribute

### ISSUE THREAD

**Issue by fractastical** [oma3dao/public-documentation#1]
It's not super clear from the readme if or how we want people to contribute to these documents. Obviously they went through a process through various oma3 working groups so maybe contact info on how to join those working groups at least?

**Comment by alftom** [comment]
Good point.  I'll add links to the README.  Currently the only way to contribute is to join OMA3 but this will likely change in the near future.  https://www.oma3.org/join.
